### PR TITLE
improve report msg

### PR DESCRIPTION
**Pull request by sivchari**

### CODE REVIEW

File: testdata/src/a/a_test.go [hunk 1]
```diff
@@ -17,26 +15,25 @@ func setup() {
...
-	if err := os.Setenv("a", "b"); err != nil { // want "func BenchmarkF is not using testing.Setenv"
+	os.Setenv("a", "b")                         // want "os.Setenv() can be replaced by `b.Setenv()` in BenchmarkF"
+	if err := os.Setenv("a", "b"); err != nil { // want "os.Setenv() can be replaced by `b.Setenv()` in BenchmarkF"
 		_ = err
 	}
 }
 
 func testTB(tb testing.TB) {
-	os.Setenv("a", "b") // want "func testTB is not using testing.Setenv"
+	os.Setenv("a", "b") // want "os.Setenv() can be replaced by `tb.Setenv()` in testTB"
```
Comment (ldez) [inline testdata/src/a/a_test.go]:
```suggestion
	os.Setenv("a", "b") // want "os\\.Setenv\\(\\) can be replaced by `tb\\.Setenv\\(\\)` in testTB"
```

File: testdata/src/a/a_test.go [hunk 2]
```diff
@@ -17,26 +15,25 @@ func setup() {
...
 		_ = err
 	}
 }
 
 func BenchmarkF(b *testing.B) {
 	testTB(b)
-	os.Setenv("a", "b")                         // want "func BenchmarkF is not using testing.Setenv"
-	if err := os.Setenv("a", "b"); err != nil { // want "func BenchmarkF is not using testing.Setenv"
+	os.Setenv("a", "b")                         // want "os.Setenv() can be replaced by `b.Setenv()` in BenchmarkF"
+	if err := os.Setenv("a", "b"); err != nil { // want "os.Setenv() can be replaced by `b.Setenv()` in BenchmarkF"
```
Comment (ldez) [inline testdata/src/a/a_test.go]:
```suggestion
	os.Setenv("a", "b")                         // want "os\\.Setenv\\(\\) can be replaced by `b\\.Setenv\\(\\)` in BenchmarkF"
	if err := os.Setenv("a", "b"); err != nil { // want "os\\.Setenv\\(\\) can be replaced by `b\\.Setenv\\(\\)` in BenchmarkF"
```

File: testdata/src/a/a_test.go [hunk 3]
```diff
@@ -17,26 +15,25 @@ func setup() {
...
-	env = os.Getenv("a")
 	os.Setenv("a", "b") // if -all = true, "func setup is not using testing.Setenv"
 }
 
 func TestF(t *testing.T) {
 	setup()
-	os.Setenv("a", "b")                         // want "func TestF is not using testing.Setenv"
-	if err := os.Setenv("a", "b"); err != nil { // want "func TestF is not using testing.Setenv"
+	os.Setenv("a", "b")                         // want "os.Setenv() can be replaced by `t.Setenv()` in TestF"
+	if err := os.Setenv("a", "b"); err != nil { // want "os.Setenv() can be replaced by `t.Setenv()` in TestF"
```
Comment (ldez) [inline testdata/src/a/a_test.go]:
```suggestion
	os.Setenv("a", "b")                         // want "os\\.Setenv\\(\\) can be replaced by `t\\.Setenv\\(\\)` in TestF"
	if err := os.Setenv("a", "b"); err != nil { // want "os\\.Setenv\\(\\) can be replaced by `t\\.Setenv\\(\\)` in TestF"
```

**Comment by ldez** [comment]
The `want` pattern is a regular expression, then `(`, `)`, and `.` are seen as expressions and not characters then you have to escape those characters.

**Comment by sivchari** [comment]
Thanks!  I've mistook.

**Pull request merged**